module_1/task2/index.js

```diff
@@ -0,0 +1,42 @@
+import path from 'path';
+import csv from 'csvtojson';
+import fs from 'fs/promises';
+
+const csvFilePath = path.join(__dirname, 'csv/test.csv');
+const csvToTxtFilePath = path.join(__dirname, 'cvsToTxt.txt');
```

Comment:
'csv/test.csv' and 'cvsToTxt.txt' must be function parameters. 
`const csvFilePath = path.join(__dirname,...` must be performer inside particular function

Reply:
Thanks, added commit with refactoring, let me know if i understand incorectly